Tests/StreamChatUITests/SnapshotTests/ChatMessageList/ChatMessageListVC_Tests.swift

```diff
@@ -261,3 +532,27 @@ class ChatMessageListVCDataSource_Mock: ChatMessageListVCDataSource {
         .init()
     }
 }
+
+class ChatMessageListVCDelegate_Mock: ChatMessageListVCDelegate {
```

Comment:
Should that be `private`?

Reply:
it probably makes sense to move this one to another file since it is also needed in other places

Reply:
Moving it to another file to reuse makes sense to me :)

Reply:
Done ✅